src/bindgen/writer.rs

```diff
@@ -137,7 +138,7 @@ impl<'a, F: Write> SourceWriter<'a, F> {
     }
 
     pub fn new_line(&mut self) {
-        writeln!(self.out).unwrap();
```

Comment:
`self.out.write_all(self.bindings.config.line_endings.as_str().as_bytes())` would be slightly more performant I think.

Reply:
Good point, yes. I was so annoyed by it not letting me pass the static str reference as a literal in `write!` that I didn't consider dropping down to a straight call on the file.